install/src/main/java/org/jetbrains/bsp/bazel/install/cli/CliOptions.kt

```diff
@@ -13,6 +13,8 @@ data class ProjectViewCliOptions internal constructor(
         val debuggerAddress: String?,
         val targets: List<String>?,
         val buildFlags: List<String>?,
+        val directories: List<String>?,
+        val deriveTargetsFlag: Boolean?
```

Comment:
lets keep the convention here as well
```suggestion
        val deriveTargetsFromDirectories: Boolean?
```